src/create/createRerunSuggestion.ts

```diff
@@ -30,10 +34,16 @@ export function createRerunSuggestion(options: Partial<Options>): string {
 	};
 
 	const args = Object.entries(optionsNormalized)
+		// Sort so the base is first, then the rest are compared using the `localeCompare` string method
```

Comment:
[Docs] Yeah I see why you'd add this comment in 😄 this kind of sorting is not super readable ... mentioning the `localeCompare` does seem a little redundant to me. How do you feel about a reword something like this?

```suggestion
		// Sort so the base is first, then the rest are compared alphabetically
```